cwel/style/mixin/_mxn-button.scss

```diff
@@ -0,0 +1,24 @@
+@mixin hover-effect($color) {
```

Comment:
@NathanRodger I would rename this to `button-hover-effect` as it's not used by any other components.